sdk/python/feast/client.py

```diff
@@ -493,17 +497,70 @@ def get_feature_set(
             raise grpc.RpcError(e.details())
         return FeatureSet.from_proto(get_feature_set_response.feature_set)
 
-    def list_entities(self) -> Dict[str, Entity]:
+    def list_features(
```

Comment:
Can you please add minimal examples to these two methods?

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html